GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -899,4 +899,41 @@ protected synchronized static void resetOnRestart() {
     abstract GVRRenderBundle getRenderBundle();
 
     abstract GVRRecyclableObjectProtector getRecyclableObjectProtector();
+
+    /**
+     * Capture a 2D screenshot from the position in the middle of left eye and
+     * right eye.
+     * 
+     * The screenshot capture is done asynchronously -- the function does not
+     * return the result immediately. Instead, it registers a callback function
+     * and pass the result (when it is available) to the callback function.
```

Comment:
This text is fine, as far as it goes, but I think it probably should include something about threading, something like

> The callback will happen on a background thread: It will probably not be the same thread that calls this method, and it will *not* be either the GUI or the GL thread.